front/src/applications/stdcm/components/StdcmWarningBox.tsx

```diff
@@ -48,17 +48,28 @@ const StdcmWarningBox = ({
       >
         {t(`stdcmErrors.${errorType}`)}
       </p>
+      {errorType === StdcmConfigErrorTypes.MISSING_INFORMATIONS &&
+        errorDetails?.missingFields?.length && (
```

Comment:
`missingFields` is always > 0 if `errorType === StdcmConfigErrorTypes.MISSING_INFORMATIONS`
```suggestion
      {errorType === StdcmConfigErrorTypes.MISSING_INFORMATIONS &&
        errorDetails?.missingFields && (
```

Reply:
done